test/lib/gatewaylib/hash_redeem_intent.js

```diff
@@ -20,25 +20,23 @@
 
 const GatewayLib = artifacts.require('GatewayLib');
 
-const Utils = require('../../gateway/eip20_cogateway/helpers/co_gateway_utils');
 const BN = require('bn.js');
+const Utils = require('../../gateway/eip20_cogateway/helpers/co_gateway_utils');
```

Comment:
why did you changed the sequence?

Reply:
according to eslint, node module requires must be before relative requires.